openedxscorm/scormxblock.py

```diff
@@ -195,7 +195,7 @@ def author_view(self, context=None):
 
     def student_view(self, context=None):
         student_context = {
-            "index_page_url": self.index_page_url,
+            "index_page_url": urllib.parse.unquote(self.index_page_url),
```

Comment:
We should also pass our navigation_title links through `urllib.parse.unquote` as well as tagged below.